src/api/views.py

```diff
@@ -116,3 +118,72 @@ def oai(request):
     }
 
     return render(request, template, context, content_type="application/xml")
+
+
+def kbart_tsv(request):
+    return kbart(request, delim='\t')
+
+
+def kbart(request, delim=','):
+    """
+    Produces KBART metadata output according to the spec:
+    https://doi.org/10.1080/0361526X.2017.1309826
+    @param request: the request object
+    @param delim: whether to use csv (',') or tsv ('\t') deliminators
+    @return: a rendered CSV or TSV
+    """
+    fields = ["date_first_issue_online", "num_first_vol_online",
+              "num_first_issue_online", "date_last_issue_online",
+              "num_last_vol_online", "num_last_issue_online",
+              "title_url", "first_author", "title_id", "embargo_info",
+              "coverage_depth", "notes", "publisher_name", "publication_type",
+              "date_monograph_published_print",
+              "date_monograph_published_online", "monograph_volume",
+              "monograph_edition", "first_editor",
+              "parent_publication_title_id", "preceding_publication_title_id",
+              "access_type"]
+
+    response = HttpResponse(content_type='text/csv')
+
+    writer = csv.DictWriter(response, fieldnames=fields, delimiter=delim)
+    response['Content-Disposition'] = 'attachment; filename="kbart.csv"' \
+        if delim == ',' else 'attachment; filename="kbart.tsv"'
```

Comment:
I would flip this logic around `.tsv` for delim == `\t` else `.csv`
(since your function allows arbitrary values for `delim`

Reply:
Not sure about the merit of this -- realistically, although it allows delim to be specified, it's only going to be called with either tab or comma.